### PR TITLE
set up

### PR DESCRIPTION
**Pull request by bangarangler**
@shawnantonucci

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,9 +12,97 @@
...
+			<ul>
+				<li>3</li>
+				<li>Amazon</li>
+				<li>Dev Ops</li>
+				<li>AWS</li>
+			</ul>
+		</div>
+	</section>
+
+	</div>
```
Comment (shawnantonucci) [inline index.html:101]: nice job recreating a table. But there is a better way :P https://www.w3schools.com/html/html_tables.asp

File: less/index.less [hunk 2]
```diff
@@ -53,7 +53,335 @@ table {
...
+@heading1Weight: 700;
+
+@btn-color: #808080;
+
+@heading2: 2.0rem;
+
+@heading2weight: 600;
+
+@crazy-font: 'ZCOOL KuaiLe', cursive;
+
```
Comment (shawnantonucci) [inline less/index.less:92]: awesome variables!

File: less/index.less [hunk 3]
```diff
@@ -53,7 +53,335 @@ table {
...
+// parametric mix-in
+.custom-layout(@justify, @align) {
+	  display: flex;
+	    justify-content: @justify;
+		  align-items: @align;
+
+}
+
```
Comment (shawnantonucci) [inline less/index.less:100]: nice parametric mixin

File: less/index.less [hunk 4]
```diff
@@ -53,7 +53,335 @@ table {
...
+	font-weight: @heading1Weight;
+	font-size: @heading1;
 }
 
+.secondaryHeading() {
+	font-weight: @heading2;
+	font-size: @heading2weight;
+}
+
+
```
Comment (shawnantonucci) [inline less/index.less:113]: nice use of mixins and variables

**Review by shawnantonucci — COMMENTED** [review]
great job on the project Jon my only two recomendations are to try and commit more and use alittle more detailed commit messages. Second I would try and clean up your code indenting so that its more uniform. https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html

File: index.html (hunk 1 above)
Comment (bangarangler) [reply index.html:101]: wow i feel really dumb. i struggled with that so bad. i thought i was doing something wrong. thanks for sharing that. that is so much easier. need to figure that out

**Pull request merged**